packages/runtime/container-runtime/src/gc/gcSummaryStateTracker.ts

```diff
@@ -276,38 +290,38 @@ export class GCSummaryStateTracker {
 		// If the summary was not tracked by this client, the state should be updated from the downloaded snapshot.
 		const snapshotTree = result.snapshotTree;
 		const metadataBlobId = snapshotTree.blobs[metadataBlobName];
-		if (metadataBlobId) {
```

Comment:
Are there cases where metadata is missing...?  i.e. can we assert it's not `undefined`?

Reply:
I don't think so. It was added a long time back and there shouldn't be active clients that generate summary without metadata.